assets/less/zenodo-rdm/views/item.overrides

```diff
@@ -0,0 +1,9 @@
+/***********************************************
+         Zenodo RDM Item Overrides
+***********************************************/
+
+#record-details {
+    .header {
```

Comment:
that changes every header in the record details page. Is this what we want? Can you add a screenshot with the whole page and compare it with zenodo? I would be tempted to change only the header in the sidebar as the issue was stating...

Reply:
![Screenshot from 2022-12-06 13-45-01](https://user-images.githubusercontent.com/25476209/205916318-f0b692a7-83a0-451a-9cee-f8cd42599f67.png)

Unless we are duplicating ids. The records-details id only makes reference to the section we want to change, and not to the whole page.

Reply:
checked, the `#record-details` is referring only to that section.
however I think this is too broad, since semantic ui classes are usually marked as `.ui.header`.
in addition the file misses formatting (missing semi-colon)